src/features/student/locales/fr.json

```
@@ -275,7 +275,13 @@
         }
       },
       "studentSkillView": {
-        "title": "Compétence {skill}"
+        "title": "Compétence {skill}",
+        "skillEvaluate": {
+          "title": "MON ÉVALUATION"
+        },
+        "skillProgress": {
+          "title": "MA PROGRESSION"
+        }
```

```suggestion
        "evaluation": {
          "title": "Mon évaluation"
        },
        "progress": {
          "title": "Ma progression"
        },
        "title": "Compétence {skill}"
```

Ordre alphabétique + tu pourras mettre un .toUppercase() plutôt que de mettre l'uppercase ici + t'es pas obligé de mettre skill en préfix de evaluation et progress